app/controllers/application_controller.rb

```diff
@@ -1,2 +1,29 @@
+require 'net/http'
 class ApplicationController < ActionController::API
+  def index
+    # This is currently just a passthrough to the RM-API
+
+    # Form the URL
+    urlstr = "#{ENV['EBSCO_RESOURCE_MANAGEMENT_API_BASE_URL']}/rm/rmaccounts/#{ENV['EBSCO_RESOURCE_MANAGEMENT_API_CUSTOMER_ID']}#{request.fullpath}"
```

Comment:
We actually need to use our configuration here. We'll need to have the same logic to verify the okapi headers.